nanome/api/structure/complex.py

```diff
@@ -152,6 +152,25 @@ def current_frame(self, value):
     def set_current_frame(self, value):
         self.current_frame = value
 
+    @property
+    def current_conformer(self):
+        current_mol = next((
+            mol for i, mol in enumerate(self.molecules)
+            if i == self.current_frame), None)
```

Comment:
```suggestion
        current_mol = self.current_molecule
```
Very minor thing, but couldn't this use the `current_molucule` property defined below?

Reply:
good catch, the property was added after this was written